draft-ietf-quic-transport.md

```diff
@@ -999,8 +999,10 @@ retired are considered active; any active connection ID can be used.
 An endpoint SHOULD ensure that its peer has a sufficient number of available and
 unused connection IDs.  Endpoints store received connection IDs for future use
 and advertise the number of connection IDs they are willing to store with the
-active_connection_id_limit transport parameter.  An endpoint SHOULD NOT provide
-more connection IDs than the peer's limit.
+active_connection_id_limit transport parameter.  An endpoint MUST NOT provide
```

Comment:
This text indicates that the number of "active_connection_id_limit" means the number of "stored" CIDs "for future use." It might be better to address this now. See #3200.

Reply:
@kazuho I updated the PR such that the active_connection_id_limit now included the handshake and SPA CID, which should make it a fix for #3200 now. PTAL.